install/kubernetes/cilium/charts/agent/templates/daemonset.yaml

```diff
@@ -27,7 +27,7 @@ spec:
         # gets priority scheduling.
         # https://kubernetes.io/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods/
         scheduler.alpha.kubernetes.io/critical-pod: ""
-{{- with .Values.podsAnnotations }}{{- toYaml . | nindent 8 }}{{- end }}
+{{- with .Values.podAnnotations }}{{- toYaml . | nindent 8 }}{{- end }}
```

Comment:
Need other to weigh in, as I am not sure how we capture note for upgrad for this changes. Using `--reuse-values` during upgrade might remove existing pod annotations if this field is not explicitly set in `helm upgrade`

Reply:
the above is answered by Chris :)